tests/acceptance/TestHelpers/WebDavHelper.php

```diff
@@ -911,22 +911,18 @@ public static function sanitizeUrl(?string $url, ?bool $trailingSlash = false):s
 	 * move together and/or be supported together. So a more complex
 	 * matrix could be needed here.
 	 *
-	 * @param string|int $davPathVersion
-	 * @param string|int|null $chunkingVersion
+	 * @param int|string $davPathVersion
 	 *
 	 * @return boolean is this a valid combination
 	 */
 	public static function isValidDavChunkingCombination(
-		$davPathVersion,
-		$chunkingVersion
+		int|string $davPathVersion,
```

Comment:
why `int|string`. make it either int or string

Reply:
Finally, that method won't be needed, IMO. :+1: